tests/e2e-playwright/tests/conftest.py

```diff
@@ -232,6 +240,14 @@ def service_key(request: pytest.FixtureRequest) -> str:
     return os.environ["SERVICE_KEY"]
 
 
+@pytest.fixture(scope="session")
+def template_id(request: pytest.FixtureRequest) -> str:
+    if key := request.config.getoption("--template-id"):
+        assert isinstance(key, str)
+        return key
+    return os.environ["TEMPLATE_ID"]
```

Comment:
```suggestion
    return None
```
This was for backwards compatibility we do not need that anymore